Build fails after merge of #580

Looks like we missed the changes introduced in e5786ad7ffa4734249c99bede2323a4d4cbb51ab in PR #580 

`
✓[testuser@Arch-AETB] ~/build/nvme-cli $ make
NVME_VERSION = 1.10.1.31.g1d0c7
    CC plugins/intel/intel-nvme.o
plugins/intel/intel-nvme.c: In function ‘enable_lat_stats_tracking’:
plugins/intel/intel-nvme.c:1253:7: error: too many arguments to function ‘parse_and_open’
 1253 |  fd = parse_and_open(argc, argv, desc, command_line_options, &cfg,
      |       ^~~~~~~~~~~~~~
In file included from plugins/intel/intel-nvme.c:11:
./nvme.h:84:5: note: declared here
   84 | int parse_and_open(int argc, char **argv, const char *desc,
      |     ^~~~~~~~~~~~~~
make: *** [Makefile:91: plugins/intel/intel-nvme.o] Error 1
✗2 [testuser@Arch-AETB] ~/build/nvme-cli $ 

`

--- a/plugins/intel/intel-nvme.c
+++ b/plugins/intel/intel-nvme.c
@@ -1250,8 +1250,7 @@ static int enable_lat_stats_tracking(int argc, char **argv,
 		{NULL}
 	};
 
-	fd = parse_and_open(argc, argv, desc, command_line_options, &cfg,
-			sizeof(cfg));
+	fd = parse_and_open(argc, argv, desc, command_line_options);
 
 	enum Option {
 		None = -1,

